ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/_cq_editConfig.xml

```diff
@@ -13,4 +13,14 @@
                 icon="bidRule"
                 text="Edit Rules"/>
     </cq:actionConfigs>
+    <cq:inplaceEditing
+            jcr:primaryType="cq:InplaceEditingConfig"
+            active="{Boolean}true"
+            configPath="inplaceEditingConfig"
+            editorType="plaintext">
+        <inplaceEditingConfig
+                jcr:primaryType="nt:unstructured"
+                editElementQuery=".guideFieldLabel label"
```

Comment:
this seems incorrect. there is no such class  as `guideFieldLabel`  in core components.

Reply:
Okay. From what I've researched there isn't an equivalent class in core-components since there is no common class/bem extension for components that have a title property. Please correct me if I am wrong. For now I will remove line 23.

Reply:
yes, that's correct. if it's not provided, then first block element is taken by default.
can you check if it works just by giving `label` tag to avoid ambuiguity.

Reply:
Used `label:first` to target just the title label and that works fine.